bson/src/test/unit/org/bson/codecs/pojo/PojoRoundTripTest.java

```diff
@@ -105,10 +121,19 @@ public void test() {
                         + "'child': {'_id': 'child', 'myFinalField': 10, 'myIntField': 10,"
                         + "'model': {'integerField': 42, 'stringField': 'myString'}}}"));
 
+        data.add(new TestData("BsonIgnore invalid map", new BsonIgnoreInvalidMapModel("myString"),
```

Comment:
As TypeData no longer validates types, the conventions can now ignore fields that previously would have thrown a validation error before conventions could be applied.